Makefile

```diff
@@ -166,10 +166,10 @@ $(EMILY_OPENAPI_SPEC): $(INSTALL_TARGET) $(EMILY_HANDLER_SOURCE_FILES)
 # ----------------------------------------------------
 
 devenv-no-sbtc-up:
-	docker compose -f docker/docker-compose.yml --profile default --profile bitcoin-mempool up
+	docker compose -f docker/docker-compose.yml --profile default --profile bitcoin-mempool --profile observability up
 
 devenv-no-sbtc-down:
-	docker compose -f docker/docker-compose.yml --profile default --profile bitcoin-mempool down
+	docker compose -f docker/docker-compose.yml --profile default --profile bitcoin-mempool --profile observability down
```

Comment:
can we add this also to "classic" devenv-up/down? I guess we also need to expose 9181-3 from the signers in the compose to reuse the same vector.toml

Reply:
Yeah, I'll add them. And I think I can just expose the expected endpoints, vector should fail gracefully.

Reply:
Just tested this locally, and the change in https://github.com/stacks-network/sbtc/pull/1105/commits/ff4dfa551c9a5ecf1d7aefb822d44e33d03af760 works.